qiskit/providers/aer/backends/qasm_simulator.py

```diff
@@ -467,6 +479,14 @@ def configuration(self):
         config.basis_gates = self._cached_basis_gates + config.custom_instructions
         return config
 
+    def available_methods(self):
+        """Return the available simulation methods."""
+        return self._AVAILABLE_METHODS
+
+    def available_devices(self):
+        """Return the available simulation methods."""
+        return self._AVAILABLE_DEVICES
+
```

Comment:
They were in the parent class, but since they are now deprecated in some classes (and not used in the pulse simulator) I moved them into all the subclasses